src/socketHandler/gameHandler.js

```diff
@@ -1,34 +1,108 @@
-const { startRound } = require('../services/game.service');
-const { createGame } = require('../services/gamestate.service');
+const {
+  createGame,
+  getRounds,
+  addCardPlayed,
+  setStichPlayer,
+  setNextPlayer,
+  addRound,
+  addSubround,
+  getPlayers,
+  addPrediction,
+  getPredictionsForCurrentRound, getPredictionCount,
+  getNextPlayer,
+} = require('../services/gamestate.service');
+const { startRound, getWinningCard } = require('../services/game.service');
 const { getCurrentLobby } = require('../services/lobby.service');
 const { getByWebsocket } = require('../services/user.service');
+const Card = require('../utils/card.model');
 
 const startGame = async function (socket, io) {
   const user = await getByWebsocket(socket.id);
   const lobby = await getCurrentLobby(user.uuid);
 
   createGame(lobby.lobbyid, lobby.players);
 
+  addSubround(lobby.lobbyid);
+
   const gameData = startRound(1, 3);
   io.to(lobby.lobbyid).emit('startGame', gameData);
 };
 
-const cardPlayed = function (socket, io, payload) {
+const cardPlayed = async function (socket, io, payload) {
   console.log('Card played: ', payload);
 
-  // TODO: Play a card
+  const player = await getByWebsocket(socket.id);
+  const lobby = await getCurrentLobby(player.uuid);
+  const lobbyId = lobby.lobbyid;
+
+  try {
+    const { value, suit, trump } = payload;
+    const card = new Card(suit, value);
+
+    addCardPlayed(lobbyId, player, card);
+    payload.player = player.username;
+    io.to(lobbyId).emit('cardPlayed', payload);
```

Comment:
This will also send the `trump`, which is probably not needed, but we can leave it for now.

Reply:
yes, the trump in the payload is not needed